.github/workflows/selfdrive_tests.yaml

```diff
@@ -117,6 +117,8 @@ jobs:
         restore-keys: |
           scons-${{ runner.arch }}-macos-${{ env.CACHE_COMMIT_DATE }}
           scons-${{ runner.arch }}-macos
+      # make new cache
+    - run: rm -rf /tmp/scons_cache/*
```

Comment:
we need to fix this scons dependency properly

Reply:
good example of why we need to do this properly, from latest master: https://github.com/commaai/openpilot/actions/runs/10151615366/job/28071268349